torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1230,7 +1230,9 @@ class StatelessSymbolicContext(SymbolicContext):
     This will cause fresh symbols to be allocated
     """
     dynamic_sizes: DimList[DimDynamic]
+    dynamic_strides: DimList[DimDynamic]
     constraint_sizes: DimList[DimConstraint] = None
+    constraint_strides: DimList[DimConstraint] = None
```

Comment:
It's hard to imagine a situation where people will actually want to put constraints on strides, but the symmetry is probably fine